patches/@mauron85+react-native-background-geolocation+0.6.3.patch

```diff
@@ -1,13 +1,886 @@
+diff --git a/node_modules/@mauron85/react-native-background-geolocation/android/common/VERSIONS.gradle b/node_modules/@mauron85/react-native-background-geolocation/android/common/VERSIONS.gradle
```

Comment:
can the patch be in a separate file to the existing lib patch?

Reply:
I'm not sure :) this is the first time I've used patch-package. I'll look into it

Reply:
This will do for now